script.module.ptw/lib/ptw/libraries/client.py

```diff
@@ -19,13 +19,11 @@
 
 
 import re, sys, cookielib, urllib, urllib2, urlparse
-import StringIO, HTMLParser, time, random, base64, requests
+import StringIO, HTMLParser, time, random, base64
```

Comment:
Jaki cel przyświeca usuwaniu modułu `requests`?
Jak widzę poniżej jest o wiele więcej kodu. Zdecydowana większość to obsługa połączenia, co zapewnia `requests`.